src/uefi/auth.c

```diff
@@ -1356,6 +1356,14 @@ verify_time_based_payload(UTF16 *name, size_t namesz, EFI_GUID *guid,
     free(new_data);
 
     if (!verify_status) {
+        INFO("Auth var failed.  PKCS7 was:\n");
```

Comment:
Two spaces again. It's hard to change habits, isn'it? :sweat_smile:

Reply:
lol sure is :(